src/plugins/layout.ts

```diff
@@ -411,5 +334,38 @@ export default class Layout implements IPlugin {
       });
     }
 
+    return events;
+  }
+
+  private ensureConsistency(lastAction: string): void {
+    if (config.ensureConsistency) {
```

Comment:
>ensureConsistency [](start = 15, length = 17)

I think we should switch either the flag name or the method name to disambiguate them